pommesdispatch/model_funcs/helpers.py

```diff
@@ -120,7 +118,80 @@ def convert_annual_limit(annual_limit, start_time, end_time):
             new_limit += annual_limit
 
         # Add limits for fractions of the start year and end year
-        new_limit += (annual_limit * start_year_fraction
-                      + annual_limit * end_year_fraction)
+        new_limit += (
+            annual_limit * start_year_fraction
+            + annual_limit * end_year_fraction
+        )
 
     return new_limit
+
+
+def convert_annual_costs_nominal_to_real(
+    nominal_costs, inflation_rate=1.02, year=2022
+):
+    """Convert cost values of DataFrame from nominal to real terms
+
+    Parameters
+    ----------
+    nominal_costs: :obj:`pd.DataFrame`
+        Nominal costs data in annual resolution (years = columns)
+
+    inflation_rate: :obj:`float`
+        Inflation rate
+
+    year: :obj:`int`
+        Year for which the nominal costs shall be expressed
+
+    Returns
+    -------
+    real_costs: :obj:`pd.DataFrame`
+        Real costs data in annual resolution
+    """
+    real_costs = nominal_costs.copy()
+    for column in real_costs.columns:
+        if column != "label":
+            try:
+                real_costs[column] = real_costs[column].div(
+                    inflation_rate ** (int(column) - year)
+                )
+            except TypeError:
+                msg = (
+                    "DataFrame format not as expected\n"
+                    "Except for column 'label', "
+                    "all other columns must be integer"
```

Comment:
Might be helpful to specify that the column name must be an integer, as it is the years?
Values of the rows can be floats as tehy are tho cost. Thus, no confusion is possible.

Reply:
Yes, you are right. The column name must be an integer. The dtype is probably float. I'll adjust the error message. Thank you!